srcpkgs/6cord/template

```diff
@@ -0,0 +1,19 @@
+# Template file for '6cord'
+pkgname=6cord
+version=9.5
+revision=1
+wrksrc="$pkgname-stable-$version"
+build_style=go
+go_import_path="gitlab.com/diamondburned/6cord"
+short_desc="Terminal UI for Discord"
+maintainer="Paper <paper@tilde.institute>"
+license="MPL-2.0"
+homepage="https://gitlab.com/diamondburned/6cord"
+distfiles="https://gitlab.com/diamondburned/$pkgname/-/archive/stable-$version/$pkgname-stable-$version.tar.gz"
+checksum=8ba69d154842aac4cebf85ff92e6d39d31827a43042bcccfcad3185876d5319b
+
+post_install() {
+	vdoc README.md
+	mkdir -p ${DESTDIR}/usr/share/examples/$pkgname
+	cp 6cord.toml ${DESTDIR}/usr/share/examples/$pkgname/
```

Comment:
> `vsconf()` `vsconf <file> [<name>]`
> 
> Installs file into usr/share/examples/<pkgname> in the pkg $DESTDIR. The optional 2nd argument can be used to change the file name.

the `vsconf` function can be used instead.

Reply:
thanks